samples/drivers/watchdog/boards/stm32.overlay

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 Thomas Stranger
+ * Copyright (c) 2022 STMicroelectronics
```

Comment:
This file was not touched in this PR. Why the copyright change?

Reply:
new file added, restoring the text though